wit/core.wit

```diff
@@ -0,0 +1,89 @@
+package wasmcloud:core@0.1.0
+
+interface types {
+  type link-settings =          tuple<string, string>
+  type host-env-values =        tuple<string, string>
+  type actor-links =            list<link-defination>
+  type cluster-issuers =        list<cluster-issuer-key>
+  type cluster-issuer-key =     string 
+  type capability-contract-id = string
+  type blob =                   list<u8>
+  type trace-context =          tuple<string, string>
+
+  enum log-level {
+    trace,
+    debug,
+    info,
+    warn,
+    error,
+  }
+
+  record link-defination {
+    actor-id:     string,
+    provider-id:  string,
+    link-name:    string,
+    contract-id:  string,
+    values:       option<link-settings>,
+  }
+
+  record health-check-response {
+    healthy: bool,
+    message: string,
+  }
+
+  record health-check-request { }
```

Comment:
wit supports function calls with no input, so we don't need this placeholder struct, we can just make the operation not take any input.